### PR TITLE
[T4A7][W14-B4]Abel Lim

### PR DESCRIPTION
**Pull request by abellimz**

### CODE REVIEW

**Comment by abellimz** [comment]
Ready for review

File: src/seedu/addressbook/data/person/Name.java [hunk 1]
```diff
@@ -36,6 +36,20 @@ public Name(String name) throws IllegalValueException {
...
+     * after trimming the spaces and converting to lower case
+     */
+     public boolean isSimilar(Name other) {
+    	 if (other == null) {
+    		 return false;
+    	 }
+    	 String thisName = fullName.trim().toLowerCase();
+    	 String otherName = other.fullName.trim().toLowerCase();
+    	 return thisName.equals(otherName);
+     }
```
Comment (damithc) [inline src/seedu/addressbook/data/person/Name.java:52]: Good try, but you are a bit off from TDD. For example, a TDD session starts by adding a failing test, not by adding an empty method. Please take a look at the course handouts and suggested readings given in the assignment description to get a good understanding of TDD.

**Comment by damithc** [comment]
Good try, but you are a bit off from TDD. For example, a TDD session starts by adding a failing test, not by adding an empty method. Please take a look at the course handouts and suggested readings given in the assignment description to get a good understanding of TDD.
Please close the PR after reading this comment.
